integration-tests/src/test/java/com/ca/mfaas/discoverableclient/DownloadApiIntegrationTest.java

```diff
@@ -36,6 +36,7 @@ public void shouldDoGetRequestAndDownloadZipFile() {
             expect().
             statusCode(200).
             header("Content-Disposition", "attachment;filename=sample-text.zip").
+            header("Content-Encoding", "gzip").
             when().
             get(uri)
             .then().contentType("application/zip")
```

Comment:
![image](https://user-images.githubusercontent.com/4791635/71832422-e4241480-3078-11ea-93f3-c915d0409212.png)

i believe we can also add Content-Encoding - "gzip" check as well.
.then().contentType("application/zip")
 .contentEncoding("gzip") as well